src/bot-handlers/liquidator.js

```diff
@@ -0,0 +1,72 @@
+const {
+  Finding, FindingSeverity, FindingType,
+} = require('forta-agent');
+
+// data to initialize contracts we are monitoring
+const contractData = require('./liquidator-contract-data.json');
+
+const {
+  umaEverestId,
+} = require('../../agent-config.json');
+
+const {
+  initializeContracts, checkIsExpiredOrShutdown,
+} = require('./initialization');
+
+// formats provided data into a Forta alert
+function createAlert(financialContractClient, position, price) {
+  return Finding.fromObject({
+    name: `Liquidatable UMA position on contract ${financialContractClient.address}`,
+    description: 'Position is under collateralized and can be liquidated',
+    alertId: 'AE-UMA-LIQUIDATABLE-POSITION',
+    severity: FindingSeverity.Medium,
+    type: FindingType.Degraded,
```

Comment:
If you think `FindingType.Info` would make more sense here, we could upgrade `forta-agent` to `0.0.18`